modules/_node-scripts/bin.js

```diff
@@ -40,6 +40,11 @@ const COMMANDS = {
 		parameters: '',
 		script: './tsconfig/index.mjs',
 	},
+	'setup': {
```

Comment:
This target will setup `node-scripts` making any previous step needed for the tool to be functional. For now it's just downloading the binary `sass` compiler, but we can add more things here in the future

Reply:
BTW: this will be invoked from the `install` hook of the package.